spec/files/gemfile_unparseable

```diff
@@ -0,0 +1,3 @@
+source 'https://rubygems.org'
+
+eval File.read('unavailable_file')
```

Comment:
Some Gemfiles load other files; this is something Polariscope can't handle in certain modes (e.g. when you use `Polariscope.scan` with non-local Gemfile).